wiki/Silence/en.md

```diff
@@ -1,43 +1,62 @@
+- Take part in [beatmap discussions](/wiki/Beatmap_discussion).
+- Edit profile details, change the avatar, cover and userpage.
+- Submit and update beatmaps.
+- Partake in [multiplayer](/wiki/Client/Interface/Multiplayer) games.
 
 ::: Infobox
 ![](img/silence-ingame.png "Silence warning in the chat")
 :::
 
-After a user is silenced, an alert will be displayed in place of the chat input field, saying when the silence wears off. The silence record can also be observed at the bottom of the profile on the website, from which it will disappear after 28 days.
+Once a silence is issued to a user, all of their messages in chat are removed and an alert will be displayed in the place of the chat input field containing the duration of the active silence.[^chat-cleanup]
+
+## Durations
 
-## Appealing a silence
+The duration of a silence starts at 5 minutes and doubles for each subsequent offence, up to a maximum of 28 days, but the initial duration may vary depending on the offence committed and the user's prior history. The factors considered when determining the length of a silence include:
 
-If you believe you were silenced unfairly, contact the [account support team](/wiki/People/Account_support_team#accounts@ppy.sh) at [accounts@ppy.sh](mailto:accounts@ppy.sh) to contest the silence. Make sure to do it from the email linked to your osu! account; mention your osu! username, as well as the approximate date and time when the silence was handed out.
+- **Atmosphere:** The current chat environment's state.
+- **Frequency:** Repetitiveness regardless of previous silences given.
+- **History:** Records of previous infringements.
+- **Severity:** The seriousness of the offence committed.
+
+In some cases, users may receive a final warning before an account restriction is placed, providing an opportunity to cease the offending behaviour.
 
 ## Common reasons for silences
 
-*Caution: Engaging in "inappropriate behaviour" in private messages is not against the rules, unless said inappropriate behaviour is being delivered to unwilling or unconsenting recipients. If reported, appropriate punishments will be placed.*
+Here are some common reasons why users get silenced in public chat, but not limited to:
+
+- **Spamming or flooding:** Self-explanatory.
+- **Caps abuse:** Chatting with all uppercase letters.
+- **Inappropriate behavior/conversation:** The public chat is not a place for such discussions.
+- **Racism:** Discrimination or hatred based on race, religion, gender, or sexual orientation.
+- **Advertising:** Promoting products or services. This includes Discord invite links and streaming links such as Twitch and YouTube.
+- **Unwanted content:** Websites containing piracy, screamers, referrals and the like. 
+
+Silences may also be given to users for other reasons such as but not limited to:
+
+- **Inappropriate beatmap submission:** Details such as metadata, tags and background image.
```

Comment:
Needs a serial comma (and inappropriate difficulty names are not that uncommon).
```suggestion
Silences may also be given to users for other reasons, such as (but not limited to):

- **Inappropriate beatmap submission:** Details such as metadata, tags, difficulty names, and background images.
```